### DISCUSSION TITLE
Parameter selection of samples enriched with beads？

### DISCUSSION THREAD

**Discussion by pokhf123** [vdemichev/DiaNN#1315] [General]
Thank you very much for the excellent DIANN software that brings us the convenience of data analysis!
We have recently used an experimental procedure like biotin magnetic bead enrichment, where theoretically the control sample should have a very low intensity of protein and a small amount, while the experimental sample should have a relatively high quantity and intensity of protein. Can we import raw from the control group and the experimental group together into DIANN for analysis? In this case, 1. What should be chosen in cross-run normalization? RT-dependent? Rt & Singal-DEP? Or something else? 2. What should I choose in Library generation? Smart profiling? Or something else? 3. Is QuantUMS(high precision) selected in the Quantification strategy? 4. Should I choose Double-pass NNs in Machine learning? Thank you again.

**Comment by vdemichev** [comment]
Hi,

Yes, you can analyses all runs all together.

> 
What should I choose in Library generation? 

Always best to keep settings default unless specific reason to change https://github.com/vdemichev/DiaNN?tab=readme-ov-file#changing-default-settings.

It's quite an open question what to do with AP-MS normalisation-wise. I would compare DIA-NN's RT-dependent with normalisation disabled, in each case maybe some custom normalisation on top in R, with some biological readout. 

Best,
Vadim
